eng/pipelines/tasks/BinSkim.yml

```diff
@@ -0,0 +1,15 @@
+steps:
+- task: DownloadPipelineArtifact@2
```

Comment:
Remove. 

You are already downloading the artifact in APIScan.yml. Do you need to download the artifact again?